docs/contributors/develop.md

```diff
@@ -1,11 +1,15 @@
 # Developer Contributions
 
-Please see [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md) for technical details on how to setup and make contributions to the Gutenberg repository.
+Please also see [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md) for general information about contributions to the Gutenberg repository.
 
 The following resources offer additional information for developers who wish to contribute to Gutenberg:
 
+* [Getting Started](/docs/contributors/getting-started.md).
+* [Git Workflow](/docs/contributors/git-workflow.md).
 * [Coding Guidelines](/docs/contributors/coding-guidelines.md) outline additional patterns and conventions used in the Gutenberg project.
-* [Gutenberg Block Grammar](/docs/contributors/grammar.md)
```

Comment:
I don't know if it was intentional, but I like the reordering to have the more advanced topics arranged further down the list.

Reply:
Yes, my idea was to put at the top the most useful documents in the context of new contributions.